ui/webui/src/components/HeaderKebab.jsx

```diff
@@ -135,40 +130,35 @@ export const HeaderKebab = ({ reportLinkURL, isConnected }) => {
         setIsReportIssueOpen(true);
     };
 
-    const dropdownItems = [
-        <DropdownItem id="about-modal-dropdown-item-about" key="about" onClick={handleAboutModal}>
-            {_("About")}
-        </DropdownItem>,
-        <DropdownItem id="about-modal-dropdown-item-report" key="report issue" onClick={handleReportIssue}>
-            {_("Report Issue")}
-        </DropdownItem>,
-    ];
     return (
         <>
             <Dropdown
-              position={DropdownPosition.right}
               onSelect={onSelect}
-              toggle={
-                  <KebabToggle
-                    id="toggle-kebab"
-                    onToggle={(_event, isOpen) => onToggle(isOpen)}
-                  />
-              }
               isOpen={isOpen}
-              isPlain
-              dropdownItems={dropdownItems}
-            />
+              popperProps={{ position: "right" }}
+              toggle={toggleRef => <MenuToggle ref={toggleRef} variant="plain" onClick={onToggle} isExpanded={isOpen} style={{ position: "absolute", right: "0" }}>
```

Comment:
I'd maybe expand the MenuToggle on multiple lines for better readability here, but don't insist.